instance/tests/integration/integration_instance.py

```diff
@@ -151,7 +151,11 @@ def test_activity_csv(self):
         #   Appserver IP,Owner Emails,Unique Hits,Total Users,Total Courses,Age (Days)
         #   149.202.188.116,"['brandon@opencraft.com', 'x@bdero.me']",4,7,1,0
 
-        self.assertEqual('Appserver IP,Owner Emails,Unique Hits,Total Users,Total Courses,Age (Days)', out_lines[0])
+        self.assertEqual(
+            '"Appserver IP","Internal LMS Domain","Name","Contact Email","Unique Hits","Total Users","Total Courses",'
+            '"Age (Days)"',
+            out_lines[0]
+        )
```

Comment:
I went ahead and added a check to make sure the email and instance name is in the second line - and also a check to make sure nothing is returning as N/A. At some point we might want to do fuzzy checks on the stats and make this test more exact if we start relying on this more in the future.